en/setup/getting_started.md

```diff
@@ -4,11 +4,78 @@ Awesome! You've downloaded a marketing automation tool. That's a great first ste
+
+The options for backing up existing tables only apply if you are installing multiple Mautic instances in the same database - in which case you may want to back up any tables that already exist using the same prefix, using the backup table prefix bak_.  This will rename any existing tables it finds using the same prefix of mau_ so that they will become bak_mau_. It is safe to leave this enabled, just in case there are tables already existing!
+
+This step may take a few minutes to complete, as the database is created and populated.
+
+### Admin user
+The next step after creating the database is to set up your administrator account.  This will be the account you use to log into Mautic for the first time.
+
+![](/setup/media/admin-user.png)
+
+Enter a username, password, first and last name and an email address, and the account will be created for you.
+
+### Email configuration
+
+The next step allows you to configure the way in which email is sent from your Mautic instance.  
+
+![](/setup/media/email-config.png)
+
+If you are testing, whether locally or on a live server, it is highly recommended to make use of [Mailhog](https://github.com/mailhog/MailHog) or similar to capture all outgoing emails, until you are ready to send to real recipients. Otherwise, in a production environment you will need to choose whether you want to send email immediately, or queue email (and process it when the Cron job runs), and provide the appropriate server, port, encryption type, credentials and authentication mode for the systems you wish to use.
+
+###Admin login
+Now you have completed the setup process, you will be directed to the back-end login screen at mydomain.com/s/login - this is where you log into Mautic to manage your instance, so it's worth bookmarking the login page for future reference.
+
 
 ## Step 2: Add Cron Jobs
 
-Once you've installed Mautic you will need to create a few standard cron jobs to have your software process various tasks. These cron jobs can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the cron jobs you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed.)
+Once you've installed Mautic you will need to create a few standard cron jobs to have your software process various tasks. These cron jobs can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the cron jobs you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)
```

Comment:
```suggestion
Once you've installed Mautic you will need to create a few standard [cron jobs][cron] to have your software process various tasks. These [cron jobs][cron] can be created through a cPanel or added through command line. If you are unfamiliar or uncomfortable with this step then we'd recommend asking in the forums or in the live Slack chat. Here is a list of the [cron jobs][cron] you'll need to create. (Please note: "/path/to/mautic..." will reflect the directory where your version of Mautic is installed, and you may need to replace 'php' with the path to PHP on your server. Ask your hosting provider if you are not sure.)

[cron][cron_jobs.md]

```

### Question

Is `[cron][cron_jobs.md]` or `[cron][cron_jobs.html]` the correct link here?  ie does `gitbook` translate a `.md` file link to the post-rendered `.html`

Reply:
### Answer

    [cron][cron_jobs.html]